### PR TITLE
feat: GEOT-118: add expanding textarea component

### PR DESCRIPTION
**Pull request by mariusz-kabala**

### CODE REVIEW

**Comment by mariusz-kabala** [comment]
retest this please

File: jest.coverage.ignore.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ const PERMANENT_IGNORE = [
   './pages/_app.tsx',
   './ui/index.ts',
   './packages/styleguide/src/StyleGuide.tsx',
+  './projects/gtms-frontend/packages/ui/Forms/index.ts',
```
Comment (ra8ga) [inline jest.coverage.ignore.js]: is it right path?

File: jest.coverage.ignore.js [hunk 2]
```diff
@@ -16,6 +17,7 @@ const TEMPORARY_IGNORE = [
   './packages/app-main/src/pages/rules/index.tsx',
   './packages/app-main/src/providers/Rules/index.tsx',
   './packages/app-main/src/components/rules/AcceptRulesButton/index.tsx',
+  './projects/gtms-frontend/packages/ui/Forms/ExpandingTextarea/index.tsx',
```
Comment (ra8ga) [inline jest.coverage.ignore.js]: is it right path?

File: jest.coverage.ignore.js (hunk 1 above)
Comment (mariusz-kabala) [reply jest.coverage.ignore.js]: yes, this is export file, there is nothing to cover there

File: jest.coverage.ignore.js (hunk 2 above)
Comment (mariusz-kabala) [reply jest.coverage.ignore.js]: its pretty tricky to cover this component with unit tests, so to not block you I will do it in a separate PR

**Pull request merged**